src/markdown-pages/install-with-kurl/index.md

```diff
@@ -125,38 +125,37 @@ A list of releases can be found on the [kURL Releases](https://github.com/replic
 *NOTE: Version pinning is supported as of release `v2021.05.07-0`.*
 
 ## Latest
-`latest` is a specific distro that is managed by the team at Replicated. This installer provides the most recent version of several add-ons and the most recent version of Kubernetes that kURL supports. Currently the spec for `latest` is:  
+`latest` is a specific distro that is managed by Replicated. This installer typically provides the most recent version of several add-ons and the most recent version of Kubernetes that kURL supports. The `latest` version of an add-on is the most recent version that Replicated is confident will continue to work when you upgrade to it.
```

Comment:
@AmberAlston  Does "distro" mean "distribution"? I wasn't 100% sure so I didn't change it. But we try to avoid using abbreviated terms, like "spec" and use "specification" instead. So I recommend spelling out what it meant here.

Reply:
Correct on distro meaning. Changes work for me. Please take a final look and approve if you think it's good to go.